src/Components/Notes/Notes.jsx

```diff
@@ -34,6 +35,7 @@ export default function Notes() {
   const {user} = useAuth0()
   const isMobile = useIsMobile()
 
```

Comment:
@pablo-mayrgundter this is a problem, selectedNote does not get set when it is accessed through the URL
Here are the logs
<img width="1178" alt="image" src="https://github.com/bldrs-ai/Share/assets/3433606/75bae2b2-e9e3-4b58-a6b2-06c83fbc10da">

it is null null null and then it is undefined